node/src/chain_spec.rs

```diff
@@ -315,6 +315,7 @@ fn devnet_genesis(
 	id: ParaId,
 ) -> serde_json::Value {
 	use pop_runtime_devnet::EXISTENTIAL_DEPOSIT;
+	let asset_hub = ismp_parachain::ParachainData { id: 1000, slot_duration: 6000 };
```

Comment:
Fine for devnet, but worth noting that AH on Paseo is still 12s so would imagine that the vec of parachains will need to be taken as a parameter in future. No change required now tho.